hudi-flink-datasource/hudi-flink/src/test/java/org/apache/hudi/sink/TestWriteCopyOnWrite.java

```diff
@@ -188,7 +188,7 @@ public void testUpsertWithDelete() throws Exception {
   @Test
   public void testInsertWithMiniBatches() throws Exception {
     // reset the config option
-    conf.setDouble(FlinkOptions.WRITE_BATCH_SIZE, 0.0008); // 839 bytes batch size
+    conf.setDouble(FlinkOptions.WRITE_BATCH_SIZE, 0.00008); // 839 bytes batch size
```

Comment:
Just for my own education: why did we need to change this?

Reply:
Hey, this is a good point! Previously, I thought the object sizes were being overestimated. But, then I actually looked into the records and I realized this JOL CLassLayout is under-estimating. It's giving a shallow size. So, we have got to use GraphLayout to get the deep size (including the size of objects that are being referenced from the current object). I'll update the patch soon after doing some more tests.